src/leaflike/bookmarks/db.clj

```diff
@@ -0,0 +1,37 @@
+(ns leaflike.bookmarks.db
+  (:require [clojure.java.jdbc :as jdbc]
+            [leaflike.utils :as utils]
+            [leaflike.config :refer [db-spec]]
+            [honeysql.core :as sql]
+            [honeysql.helpers :as helpers]))
+
+(defn create-bookmark
+  [body]
+  (utils/add-created-at body)
```

Comment:
This will not have any effect on the `body` map

Reply:
It is an `assoc` function. Still?

Reply:
<img width="545" alt="screen shot 2017-12-14 at 11 47 33 am" src="https://user-images.githubusercontent.com/6064926/33978218-a9668676-e0c4-11e7-93b2-06bce78619f9.png">

It works as intended.